packages/mantle/src/components/command/command.tsx

```diff
@@ -0,0 +1,438 @@
+ *       <Command.Item>
+ *         <span>Profile</span>
+ *       </Command.Item>
+ *     </Command.Group>
+ *   </Command.List>
+ * </Command.Dialog>
+ */
+const CommandItem = forwardRef<
+	ComponentRef<"div">,
+	ComponentPropsWithoutRef<typeof CommandPrimitive.Item>
+>(({ className, ...props }, ref) => (
+	<CommandPrimitive.Item
+		ref={ref}
+		data-slot="command-item"
+		className={cx(
+			"data-[selected=true]:bg-popover-hover [&_svg:not([class*='text-'])]:text-muted relative flex cursor-pointer items-center gap-2 rounded-sm px-2 py-1.5 text-sm outline-hidden select-none data-[disabled=true]:pointer-events-none data-[disabled=true]:opacity-50 [&_svg]:pointer-events-none [&_svg]:shrink-0 [&_svg:not([class*='size-'])]:size-4",
+			className,
+		)}
+		{...props}
+	/>
+));
+CommandItem.displayName = "CommandItem";
+
+/**
+ * The shortcut component for the Command. It provides the shortcut for the command palette.
+ *
+ * @see https://mantle.ngrok.com/components/command#api-command-shortcut
+ *
+ * @example
+ * ```tsx
+ * <Command.Dialog>
+ *   <Command.Input placeholder="Type a command or search..." />
+ *   <Command.List>
+ *     <Command.Empty>No results found.</Command.Empty>
+ *     <Command.Group heading="Suggestions">
+ *       <Command.Item>
+ *         <span>Calendar</span>
+ *       </Command.Item>
+ *     </Command.Group>
+ *     <Command.Separator />
+ *     <Command.Group heading="Settings">
+ *       <Command.Item>
+ *         <span>Profile</span>
+ *         <Command.Shortcut>⌘,</Command.Shortcut>
+ *       </Command.Item>
+ *     </Command.Group>
+ *   </Command.List>
+ * </Command.Dialog>
+ */
+const CommandShortcut = forwardRef<
+	ComponentRef<"span">,
+	ComponentPropsWithoutRef<"span">
+>(({ className, ...props }, ref) => (
+	<span
+		ref={ref}
+		data-slot="command-shortcut"
+		className={cx("text-muted ml-auto text-xs tracking-widest", className)}
+		{...props}
+	/>
+));
+CommandShortcut.displayName = "CommandShortcut";
+
+/**
+ * The command component for the Command. It provides the command for the command palette.
+ *
+ * @see https://mantle.ngrok.com/components/command#api-command
+ *
+ * @example
+ * ```tsx
+ * <Command.Dialog>
+ *   <Command.Input placeholder="Type a command or search..." />
+ *   <Command.List>
+ *     <Command.Empty>No results found.</Command.Empty>
+ *     <Command.Group heading="Suggestions">
+ *       <Command.Item>
+ *         <span>Calendar</span>
+ *       </Command.Item>
+ *     </Command.Group>
+ *     <Command.Separator />
+ *     <Command.Group heading="Settings">
+ *       <Command.Item>
+ *         <span>Profile</span>
+ *       </Command.Item>
+ *     </Command.Group>
+ *   </Command.List>
+ * </Command.Dialog>
+ */
+const Command = {
+	Root: CommandRoot,
+	Dialog: CommandDialog,
+	Input: CommandInput,
+	List: CommandList,
+	Empty: CommandEmpty,
+	Group: CommandGroup,
+	Item: CommandItem,
+	Shortcut: CommandShortcut,
+	Separator: CommandSeparator,
```

Comment:
It's annoying (I know) but we'll need to inline the jsdocs for the components on the export entries else you won't get intellisense